2019-04-12-tor-onion-services-available-again.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Qubes Tor onion services are available again!"
+categories: announcements
+author: Andrew David Wong
+---
+
+We previously announced that the [Qubes Tor onion services were no
+longer being maintained][orig-onion-ann] due to lack of resources.
+However, [Unman] generously agreed to bring them back, and they're now
+available once again!
+
+Here are the new onion service URLs:
+
+```
+Website:   www.qubesosfasa4zl44o4tws22di6kepyzfeqv3tg4e3ztknltfxqrymdad.onion
+Yum repo:  yum.qubesosfasa4zl44o4tws22di6kepyzfeqv3tg4e3ztknltfxqrymdad.onion
+Deb repo:  deb.qubesosfasa4zl44o4tws22di6kepyzfeqv3tg4e3ztknltfxqrymdad.onion
+ISOs:      iso.qubesosfasa4zl44o4tws22di6kepyzfeqv3tg4e3ztknltfxqrymdad.onion
+```
+
+Soon, you will be able to get the new, correct repo definitions just by
+updating dom0 and your TemplateVMs. However, if you can't wait, please
```

Comment:
It still isn't explained *what you need to do "if you cant wait". The Whonix page is just about switching to onion repositories that are already set up.

I suggest:

However, if you can't wait, edit the repository definitions to include the new onion addresses. 
You need to make sure that **every** line that contains an onion address uses the appropriate **new** address above.
In dom0, edit /etc/yum.repos.d/qubes-dom0.repo and /etc/yum.repos.d/qubes-templates.repo.
In Fedora templates, edit /etc/yum.repos.d/qubes-r4.repo.
Every onion address should look like this:
#baseurl http://yum.qubesosfasa4zl44o4tws22di6kepyzfeqv3tg4e3ztknltfxqrymdad.onion/..........

In Debian templates, edit /etc/sources.list.d/qubes-r4.list and check the entries in the "Qubes Tor updates repositories" section.
A sample entry would look like this:
#deb [arch=amd64] http://deb.qubesosfasa4zl44o4tws22di6kepyzfeqv3tg4e3ztknltfxqrymdad.onion/r4.0/vm stretch main

You can then enable updates using the new onion repositories by commenting out the clearnet definitions, and removing the comment from the start of the lines containing the new onion definitions.
There are step-by-step instructions on how to do this in the Whonix wiki:

Reply:
Updated.